ts/webui/src/static/function.ts

```diff
@@ -4,6 +4,17 @@ import { IContextualMenuProps } from '@fluentui/react';
 import { MANAGER_IP } from './const';
 import { MetricDataRecord, FinalType, TableObj, Tensorboard } from './interface';
 
+function getPrefix(): string | undefined {
+    const pathName = window.location.pathname;
+    let newPathName = pathName;
+
+    if (pathName.endsWith('/oview' || '/detail' || '/experiment')) {
+        newPathName = pathName.replace('/oview' || '/detail' || '/experiment', '');
```

Comment:
What if the prefix is `/experiment-number-1`? Will it be mistakenly replaced?

Reply:
it doesn't support prefix with '/' on the head.
![image](https://user-images.githubusercontent.com/61399850/118455573-ae427600-b72b-11eb-98f8-10994ee510e8.png)